src/main/java/de/cityfeedback/logging/LoggingAspect.java

```diff
@@ -0,0 +1,42 @@
+package de.cityfeedback.logging;
```

Comment:
kannst du die aspects noch in das richtige package verschieben? Unter feedbackverwaltung.infrastructure.aspects